tests/integration/relation_tests/test_charm_relations.py

```diff
@@ -15,7 +15,12 @@
 DATABASE_METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 PORT = 27017
 DATABASE_APP_NAME = DATABASE_METADATA["name"]
-APP_NAMES = [APPLICATION_APP_NAME, DATABASE_APP_NAME]
+FIRST_DATABASE_RELATION_NAME = "first-database"
+SECOND_DATABASE_RELATION_NAME = "second-database"
+MULTIPLE_DATABASE_CLUSTERS_RELATION_NAME = "multiple-database-clusters"
+ALIASED_MULTIPLE_DATABASE_CLUSTERS_RELATION_NAME = "aliased-multiple-database-clusters"
+ANOTHER_DATABASE_APP_NAME = "another-database"
+APP_NAMES = [APPLICATION_APP_NAME, DATABASE_APP_NAME, ANOTHER_DATABASE_APP_NAME]
```

Comment:
If these constants are going to be reused, we could open a pr to data-platform-libs to make them constants in test_charm.py, and you could import them from there.